src/api/downloadFile.android.js

```diff
@@ -15,8 +15,12 @@ export default (url: string, auth: Auth) =>
       mime: 'text/plain',
       title: url.split('/').pop(),
     },
-  }).fetch('GET', url, {
-    'Content-Type': 'application/x-www-form-urlencoded; charset=utf-8',
-    'User-Agent': userAgent,
-    Authorization: getAuthHeader(auth.email, auth.apiKey),
-  });
+  })
+    .fetch('GET', url, {
+      'Content-Type': 'application/x-www-form-urlencoded; charset=utf-8',
+      'User-Agent': userAgent,
+      Authorization: getAuthHeader(auth.email, auth.apiKey),
+    })
+    .then(res => {
```

Comment:
Here you would have just awated the 'res' and then would do a `return res.path()`.